liblangutil/SourceReferenceExtractor.h

```diff
@@ -49,10 +49,11 @@ struct SourceReference
 	int endColumn = {-1};     ///< Highlighting range-end of text field.
 
 	/// Constructs a SourceReference containing a message only.
-	static SourceReference MessageOnly(std::string _msg)
+	static SourceReference MessageOnly(std::string _msg, std::string _sourceName = {})
```

Comment:
Let me know if you think it's unbearable to have something called ``MessageOnly`` optionally handle a source name as well, then I'll rename stuff :-).